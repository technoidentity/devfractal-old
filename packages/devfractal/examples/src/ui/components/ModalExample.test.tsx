import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { ModalExample } from './ModalExample'

test('Modal Example', () => {
  const component: ReactTestRenderer = renderer.create(<ModalExample />)
  expect(component.toJSON()).toMatchSnapshot()
})
