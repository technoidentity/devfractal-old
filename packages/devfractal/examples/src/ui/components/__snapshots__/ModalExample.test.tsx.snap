// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Example 1`] = `
Array [
  <section
    className="section"
  >
    <div
      className="control"
    >
      <button
        className="button is-primary is-large"
        onClick={[Function]}
        type="button"
      >
        Launch Media Modal
      </button>
    </div>
  </section>,
  <section
    className="section"
  >
    <div
      className="control"
    >
      <button
        className="button is-primary is-large"
        onClick={[Function]}
        type="button"
      >
        Launch Image Modal
      </button>
    </div>
  </section>,
  <section
    className="section"
  >
    <div
      className="control"
    >
      <button
        className="button is-primary is-large"
        onClick={[Function]}
        type="button"
      >
        Launch Card Modal
      </button>
    </div>
  </section>,
]
`;
